src/calibre/gui2/__init__.py

```diff
@@ -1680,12 +1680,11 @@ def make_view_use_window_background(view):
 
 
 def timed_print(*a, **kw):
-    if not DEBUG:
-        return
-    from time import monotonic
-    if not hasattr(timed_print, 'startup_time'):
-        timed_print.startup_time = monotonic()
-    print(f'[{monotonic() - timed_print.startup_time:.2f}]', *a, **kw)
+    if DEBUG:
+        from time import monotonic
+        if not hasattr(timed_print, 'startup_time'):
+            timed_print.startup_time = monotonic()
+        print(f'[{monotonic() - timed_print.startup_time:.2f}]', *a, **kw)
```

Comment:
This in particular is just changing

```python
if not X:
    return
Y
```
to
```python
if X:
    Y
```

and I call either spam or noise.